docs/istio.md

```diff
@@ -2,7 +2,20 @@
 
 The SMI Metrics Extension APIService makes use of prometheus metrics exposed by Istio to resopond to metric queries like p90, request counts, etc.
 
-There is a configuration change required on the Istio metrics for SMI Metrics to work, as the default metrics dosen't have enough information to relate thosoe metrics with that of Kubernetes resources.
+## Out of the Box Installation
+
+As Istio allows metrics to be configurable, SMI with Istio can be installed by directly running
+
+```bash
+helm template chart --set adapter=istio | k apply -f -
+```
+This installs the necessary instances, handlers and rules for Istio to emit those metrics along with SMI-Metrics APIServer.
+
+## Using the default Istio Metrics
```

Comment:
Should probably note that the chart requires `istio` is already installed and running.

Reply:
Noted